docs/multi-cluster-install.md

```diff
@@ -57,12 +57,13 @@ If you have `jq` and `base64` available then this one-liners will pull all CA ce
 `${HOME}/.kube/config` and place then in a file named `ca.pem`.
 
 ```shell
-kubectl config view -o json --raw  | jq -r '.clusters[].cluster["certificate-authority-data"]' | base64 -d > ca.pem
+# replace CLUSTERNAME with the name of the cluster according to your ${HOME}/.kube/config
+kubectl config view -o json --raw  | jq -r '.clusters[] | select(.name=="CLUSTERNAME").cluster["certificate-authority-data"]' | base64 -d > ca.pem
```

Comment:
Can this be added separately so that for user who have multi-cluster in their kubeconfig can follow this command?

I think we want the command to be working by default for most of single-cluster kubeconfig, but if you have multi-cluster then you can follow advanced command.